Tests/L2Tests/patches/RDKTV-35484-DeviceIdentification.patch

```diff
@@ -0,0 +1,119 @@
+diff --git a/interfaces/Ids.h b/interfaces/Ids.h
```

Comment:
Please add a header giving the source of the patch i.e. any new or changed code here.
Please explain why the patch is changing a copyright from RDK Management to Metrological?  The copyright gives the owner of the code; it would be unusual for this to change.